### ISSUE TITLE
node became unavailable through kubernetes deployment

### ISSUE THREAD

**Issue by rawdaGastan** [threefoldtech/grid_weblets#857]
### Description

When deploying a K8s Cluster with more than 2 workers for the same nodes : node became unavailable through the deployment.

### Steps to reproduce

- Deploy a K8s Cluster.
- Add 3 workers on the same node.

### Logs/Alerts

![issue3](https://user-images.githubusercontent.com/47260239/175820331-70f7bbfa-bea9-4017-911e-70feb2c2232d.png)

**Comment by rawdaGastan** [comment]
It will be solved in the next release of the grid client 1.3.6

**Comment by mohamedamer453** [comment]
Verified on devnet.

I tried to deploy a k8s cluster with a master and 3 workers on the same node with the default configuration so each of them had a 100 GB disk (a total of 400 GB) and the deployment failed with the following message.

```
Node 33 doesn't have enough resources: sru=431593881600, mru=4294967296
```
![image](https://user-images.githubusercontent.com/101194226/177322754-b5a8ba29-1aa2-4d9e-b01c-560de75ffce8.png)

and that's expected since the node didn't have enough `SRU` by checking from the explorer.

and when i changed the disks to 10 GB each the deployment was successful.

![image](https://user-images.githubusercontent.com/101194226/177325851-7dca881a-5868-46f6-98f6-6c47a97f75ac.png)

![image](https://user-images.githubusercontent.com/101194226/177336963-e686035f-3c52-4c22-bf4c-8a63522166a4.png)
